### ISSUE TITLE
"Closest surrounding pair (tree-sitter)" not working anymore in git context

### ISSUE THREAD

**Issue by karthago1** [helix-editor/helix#10651]
### Summary

Since we have git highlighting, I think this bug is introduced

Just create a file named "COMMIT_EDITMSG" and paste the following text

```
changelog:
383aa7b (HEAD -> hich/ethernet) commit 1
e10a36a commit 2
```

go inside the parentheses and press "mam" in normal mode. result: No thing is selected.

Repeat the same steps in a new buffer and the selection will be successful

here is a video
https://asciinema.org/a/PpvQrhXxDVg7EihKWGZAfqRla

### Platform

Linux

### Terminal Emulator

kitty

### Installation Method

source

### Helix Version

24.03-147-g752ed8eb

**Comment by woojiq** [comment]
I'm not sure if this is a bug. tree-sitter for git does not define a node for pairs (like markdown tree-sitter). Type `:tree-sitter-subtree`. `mm` doesn't work either.

**Comment by pascalkuthe** [comment]
Yeah this behaves as expected. Altough for some TS grammars (basically anything that is not a proper language) we may want to switch back to plaintext (maybe with a per language setting) but it could also just be fine to require the use of the more explicit plaintext bindings (`mi(`)

**Comment by karthago1** [comment]
the result of `:tree-sitter-subtree` is

```
(message
     (commit))
```

`mi(` works fine.

> we may want to switch back to plaintext (maybe with a per language setting)

thank you for the tip. I just disabled the the highlighting by editing `languages.toml`
```
[[language]]
name = "git-commit"
file-types = []
```

now I have the same behavior as the released version. (it looks not as nice as the highlighted version, but more functional)

should I close this issue. since it is not a bug in helix itself, but in the tree sitter definition of git?
